### ISSUE TITLE
event-processors with kafka

### ISSUE THREAD

**Issue by DefunctExodus** [karimra/gnmic#263]
I have been playing with the event processors and It appears that it doesn't seem to work correctly for the kafka output

This works as expected 
```
  stdout:
    type: file
    file-type: stdout
    format: event
    event-processors:
      - trim-name-prefix
 ```

however this just seems to not work, nothing gets trimmed
```
  kafka-cluster-telemetry:
    type: kafka
    address: "some-brokers:9093"
    topic: "some-topic"
    format: event
    event-processors:
      - trim-name-prefix
 ```

I am only running a single output at a time in this scenario

**Comment by hellt** [comment]
Hi @DefunctExodus 
yes, sorry, it was a typo on our (darn `-` vs `_`)
it was fixed today in #262 

If you want to continue your testing, you can either change the `event-processors` line in your config for kafka output to `event_processors`, or clone the master branch and build the gnmic from it and use the proper name with `-` afterwards

**Comment by DefunctExodus** [comment]
Oh perfect, I will just modify that for now, Thanks!
